packages/synapse-react-client/mocks/entity/mockDataset.ts

```diff
@@ -10,7 +10,7 @@ import { MOCK_USER_ID, MOCK_USER_ID_2 } from '../user/mock_user_profile'
 import { MockEntityData } from './MockEntityData'
 import mockProjectEntityData from './mockProject'
 
-const MOCK_DATASET_ENTITY_ID = 'syn234'
+const MOCK_DATASET_ENTITY_ID = `syn60000`
```

Comment:
It was a bit easier to ensure there are no collisions in generated IDs if we partition these mock IDs by type, so syn10000s are projects, syn20000s are folders. 

Generating so much data will take more thought about how we manage it, so I'm not convinced it makes sense to enforce this rule with code yet.